src/praisonai/praisonai/bots/_commands.py

```
@@ -775,3 +784,217 @@ def handle_learn_command(
+        session_manager: BotSessionManager instance.
+        user_id: User ID requesting the listing.
+
+    Returns:
+        A formatted list of session identifiers, or guidance.
+    """
+    list_fn = getattr(session_manager, "list_sessions", None)
+    keys: List[str] = []
+    if callable(list_fn):
+        try:
+            keys = list(list_fn())
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)